src/puppetlabs/trapperkeeper/services.clj

```diff
@@ -22,6 +22,8 @@
   (service-context [this] "Returns the context map for this service")
   (get-service [this service-id] "Returns the service with the given service id")
   (get-services [this] "Returns a sequence containing all of the services in the app")
+  (call-service-fn [this other-service-id fn-name args default] "Calls named function in other service, returning default if that service is not included.")
+  (service-included? [this service-id] "Returns nil if service does not exist and the service if it does")
```

Comment:
I have no real beef with the `maybe-*` pattern.

I like the idea of being able to write `if-let` blocks, and I also like the idea of having a `service-included?` predicate, but I'd probably vote for two separate fns.  So, yeah, I'd vote for changing `service-included?` to just return a bool, and adding a `maybe-get-service` (or whatever other name).

Open to other people's input on this, though.